operator/src/main/resources/application.properties

```diff
@@ -41,12 +41,20 @@ ingresscontroller.max-ingress-connections=108000
 ingresscontroller.peak-connection-percentage=100
 
 # Enables haproxy option contstats in the kas ingress so that stats are reported live rather than at connection close (RFE-3007 provide a proper mechanism to enable this option).
```

Comment:
I think we should also update the comment here.

Reply:
done